rust/private/rustc.bzl

```diff
@@ -1923,3 +1958,15 @@ extra_exec_rustc_flag = rule(
     implementation = _extra_exec_rustc_flag_impl,
     build_setting = config.string(flag = True, allow_multiple = True),
 )
+
+def _per_crate_rustc_flag_impl(ctx):
+    return PerCrateRustcFlagsInfo(per_crate_rustc_flags = [f for f in ctx.build_setting_value if f != ""])
+
+per_crate_rustc_flag = rule(
+    doc = (
```

Comment:
Can you update these docs to describe the matching behavior?

Reply:
Done.